07_control_statements_branching_and_jumps/exercises/schedule.c

```diff
@@ -0,0 +1,60 @@
+// The 1988 United States Federal Tax Schedule was the simplest in recent 
+// times. It had four categories, and each category had two rates. Here is a 
+// summary (dollar amounts are taxable income):
+// Single: 15% of first $17,850 plus 28% of excess 
+// Head of Household: 15% of first $23,900 plus 28% of excess 
+// Married, Joint: 15% of first $29,750 plus 28% of excess
+// Married, Separate: 15% of first $14,875 plus 28% of excess
+// For example, a single wage earner with a taxable income of $20,000 owes 
+// 0.15 × $17,850 + 0.28 × ($20,000-$17,850). Write a program that lets the 
+// user specify the tax category and the taxable income and that then calculates
+// the tax. Use a loop so that the user can enter several tax cases. 
+#include <stdio.h>
+
+#define RATE1 0.15
+#define RATE2 0.28
+
+int main(void)
+{
+    float gross_pay;
+    float salary;
+    float net_pay;
+    float taxes;
+    char rate;
+
+    printf("Enter the number of your category:\n");
+    printf("1) Single (15%% of first $17,850 plus 28%% of excess)\n");
+    printf("2) Head of Household (15%% of first $23,900 plus 28%% of excess)\n");
+    printf("3) Married, Joint (15%% of first $29,750 plus 28%% of excess)\n");  
+    printf("4) Married, Separate (15%% of first $14,875 plus 28%% of excess)\n");
+    printf("5) quit\n");
+    while ((rate = getchar()) != '5')
+    {
+        switch (rate)
+        {
+            case '1' : gross_pay = 17850;
```

Comment:
17850, 23900 etc should be constants

TAX_BREAK_1  17850.0
TAX_BREAK_2  23900.0
...